src/signalhandler.cpp

```diff
@@ -165,11 +165,12 @@ bool CSignalUnix::setSignalHandled ( int sigNum, bool state )
 {
     struct sigaction sa;
     sigemptyset ( &sa.sa_mask );
+    sa.sa_flags = 0;
 
     if ( state )
     {
         sa.sa_handler = CSignalUnix::signalHandler;
-        sa.sa_flags = SA_RESTART;
+        sa.sa_flags |= SA_RESTART;
```

Comment:
Why?  It's set to zero.  0 | anything is the thing.  So you may as well leave this line unchanged.

Reply:
Will do if you prefer - I'm more used to seeing this with |= so you can add other flags earlier/later.

Reply:
pljones, you are right. I also agree to atsampson that usually "|=" is used for these flags. If you ask me I would prefer atsampsons implementation. But I think since this is your code, pljones, you can decide on this :-)

Reply:
Ups, it's already gone?